bigbone/src/main/kotlin/social/bigbone/api/method/OAuthMethods.kt

```diff
@@ -76,6 +80,7 @@ class OAuthMethods(private val client: MastodonClient) {
                 append("redirect_uri", redirectUri)
                 append("code", code)
                 append("grant_type", GrantTypes.AUTHORIZATION_CODE.value)
+                scope?.let { append("scope", scope.toString()) }
```

Comment:
Potential typo: In other places, where the scope is appended to the query string, we use `scopes`, but here `scope` is used. Could you please check? Thanks a lot.

Reply:
I stumbled upon this myself, but it is correct according to Mastodon's [OAuth scopes](https://docs.joinmastodon.org/api/oauth-scopes/) documentation. I can add a comment clarifying that.

Reply:
I've added a comment to the single use of "scopes" in `AppMethods.kt`.